iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/ClearTextProtocolsCheck.java

```diff
@@ -53,4 +55,9 @@ private static void checkHttpsTraffic(ContextualResource resource) {
     resource.property("supportsHttpsTrafficOnly")
       .reportIf(isFalse(), GENERAL_ISSUE_MESSAGE);
   }
+
+  private static void checkProtocols(ContextualResource resource) {
+    resource.list("protocols")
+      .reportItemIf(CheckUtils.isEqual("http"), GENERAL_ISSUE_MESSAGE);
```

Comment:
Please use the static import of the `CheckUtils.isEqual` method to make it more readable like `report item it is equal`

Reply:
Thanks, seems like i missed it!